source/includes/_application-api.md

```diff
@@ -0,0 +1,453 @@
+# Your application API
+
+The API is the way the outside world interact with your system, and any server-side application needs one.
+With Booster, you don't need to worry about it: _it is created for you_ and evolves with your code _automatically_.
```

Comment:
It's just my opinion, but it doesn't look like this sentence is saying anything useful to me. It's trying to sell me something, although I'm already reading the docs. I'd rephrase it as:

```suggestion
The API is generated automatically when you deploy your application, and it changes when needed, no additional steps required on your side.
```